exchanges/bitfinex.go

```diff
@@ -97,7 +97,7 @@ func (wrapper *BitfinexWrapper) GetOrderBook(market *environment.Market) (*envir
 				ts = new(time.Time)
 			}
 
-			orderBook.Bids = append(orderBook.Bids, environment.Order{
+			orderBook.Asks = append(orderBook.Asks, environment.Order{
```

Comment:
why changing this?

Reply:
in the first loop(line 76), you walk through bitfinexOrderBook.**Bids** but add to orderBook.**Asks** instead of orderBook.Bids 
In the second loop is the same

Reply:
I see, perfect, awesome, merging now